docker.Makefile

```diff
@@ -46,6 +46,9 @@ clean: build_docker_image
 test-unit: build_docker_image
 	docker run --rm $(ENVVARS) $(MOUNTS) $(DEV_DOCKER_IMAGE_NAME) make test-unit
 
+.PHONY: test
+test: test-unit
```

Comment:
I think `test` should be an alias for running all the test suites, so please make this: `test: test-unit test-e2e`

Reply:
@dnephin Added! 👍